cpp/include/cudf/strings/wrap.hpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/column/column.hpp>
+
+namespace cudf
+{
+namespace strings
+{
+/**
+ * @brief Inserts new-line characters (ASCII 0x0A) into each string in place of spaces, depending on `width`.  
```

Comment:
Let's use the word "wrap" in the brief, since that's intuitive...

```suggestion
 * @brief Wraps strings onto multiple lines shorter than `width` by replacing appropriate white space with new-line characters (ASCII 0x0A).  
```